### ISSUE TITLE
[img] add support for alt text

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-third-party-extensions#25]
For accessibility, there should be a way to specify alt text.

**Comment by pkra** [comment]
Closing in favor of https://github.com/pkra/mathjax-img/issues/2
